src/main/java/org/reaktivity/nukleus/kafka/internal/stream/FetchResponseDecoder.java

```diff
@@ -66,11 +66,11 @@
     private final RecordFW recordRO = new RecordFW();
     private final HeaderFW headerRO = new HeaderFW();
 
-    private final Headers headers = new Headers();
-    private final UnsafeBuffer keyBuffer = new UnsafeBuffer(EMPTY_BYTE_ARRAY);
-    private final UnsafeBuffer valueBuffer = new UnsafeBuffer(EMPTY_BYTE_ARRAY);
+    private final HeadersFW headers = new HeadersFW();
+    private final UnsafeBuffer keyBuffer = new UnsafeBuffer(BufferUtil.EMPTY_BYTE_ARRAY);
+    private final UnsafeBuffer valueBuffer = new UnsafeBuffer(BufferUtil.EMPTY_BYTE_ARRAY);
```

Comment:
Field types should be narrower as needed, i.e. `AtomicBuffer`, `MutableDirectBuffer` or `DirectBuffer`.

Reply:
Changed to DirectBuffer